src/plugins/Topup/moonpay.ts

```diff
@@ -42,12 +42,9 @@ const orderUrl = async (
   );
   const parameters = {
     apiKey: config.moonpayLiveAPIKEY,
-    // apiKey: config.moonpayTestAPIKEY,
     enabledPaymentMethods: "credit_debit_card,sepa_bank_transfer,gbp_bank_transfer",
-    currencyCode: params.selectedCryptoCurrency || undefined,
-    // walletAdress: state.selectedAddress,
-    walletAddresses: JSON.stringify({ sol: state.selectedAddress }),
-    //   colorCode,
+    defaultCurrencyCode: params.selectedCryptoCurrency || undefined,
```

Comment:
should be `currencyCode` instead of `defaultCurrencyCode`

Reply:
I changed it because
1. Phantom uses defaultCurrencyCode
2. app.tor.us uses defaultCurrencyCode
3. using currencyCode makes the user signup on moonpay first.